src/prefect/cli/events.py

```diff
@@ -23,6 +27,19 @@ class StreamFormat(str, Enum):
     text = "text"
 
 
+def get_event_subscriber_type_for_context(
+    account: bool,
+) -> Type[PrefectEventSubscriber]:
+    experimental_events_enabled = PREFECT_EXPERIMENTAL_EVENTS.value()
+    api_url = PREFECT_API_URL.value()
+
+    if experimental_events_enabled and (api_url is None or "/account" not in api_url):
```

Comment:
this `PREFECT_API_URL` is a bit hacky, but because the tests use `localhost` urls for cloud urls, this is the best way I could readily think of. happy to do something more robust here if necessary